framework/di/Container.php

```diff
@@ -349,6 +349,21 @@ public function getDefinitions()
     }
 
     /**
+     * Configures an object with the initial property values.
+     * @param object $object the object to be configured
+     * @param array $properties the property initial values given in terms of name-value pairs.
+     * @return object the object itself
+     */
+    public function configureObject($object, $properties)
+    {
+        foreach ($properties as $name => $value) {
```

Comment:
What happens if `$object` has no `$name` property? 

This case is coverage in test?

Reply:
Throws expected exception

Reply:
And what happens in this case:

```php
...
private $b;
public function setB(array $value): void{
   $this $b;
   return;
}

...
...configureObject($TestObject, ['b' => (string) 100]);
```

Reply:
the setter is called.

Reply:
How to process mismatch of type in setter witch strict types?

Reply:
It will throw an exception. That's native PHP behavior and is correct in the case.

Reply:
@samdark yes, that's what I mean.

This is expected behaviors AND this behavior is not reflecting in tests?

I can't see It 
```php
$this->expectException('\TypeError');
```
 in `tests/framework/BaseYiiTest.php` and in `tests/framework/di/ContainerTest.php`

https://3v4l.org/LitK2 and https://3v4l.org/KQFRH

Reply:
👍 would be great if you'll add it. Thanks.